scripts/build_geant4.sh

```diff
@@ -1,17 +1,22 @@
 #!/bin/bash
 
-set -e
+set -ex
 
 source ${docker_env}
 
 rm -rf ${geant4_build_dir}/bld ${geant4_install_dir}
 mkdir -p ${geant4_build_dir}/bld
 cd ${geant4_build_dir}
-wget http://geant4.cern.ch/support/source/geant4.${geant4_version}.tar.gz
+wget https://geant4.cern.ch/support/source/geant4.${geant4_version}.tar.gz --no-check-certificate
+tar_chashum=$(sha256sum ${geant4_version}.tar.gz | cut -d' ' -f1)
+if [ $geant4_shasum != $tar_chashum ]; then
```

Comment:
If we're doing the shasum thing for Geant4, would it make sense to do it for HDF5 too?

Reply:
fair point !